pkg/app/piped/planner/kubernetes/kubernetes.go

```diff
@@ -480,3 +492,51 @@ func determineVersion(manifests []provider.Manifest) (string, error) {
 
 	return b.String(), nil
 }
+
+// determineVersions decides artifact versions of an application.
+// It finds all container images that are being specified in the workload manifests then returns their names, version numbers, and urls.
+func determineVersions(manifests []provider.Manifest) ([]*model.ArtifactVersion, error) {
+	versions := []*model.ArtifactVersion{}
+	imageMap := map[string]string{}
```

Comment:
To have smaller size.

```suggestion
	imageMap := map[string]struct{}{}
```